src/NetAnalyzers/Core/Microsoft.CodeQuality.Analyzers/ApiDesignGuidelines/CancellationTokenParametersMustComeLast.cs

```diff
@@ -99,6 +100,19 @@ public override void Initialize(AnalysisContext context)
                         last--;
                     }
 
+                    // Ignore parameters that hav any of these attributes.
+                    // C# reserved attributes: https://docs.microsoft.com/dotnet/csharp/language-reference/attributes/caller-information
+                    var callerInformationAttributes = ImmutableArray.Create<INamedTypeSymbol>(
```

Comment:
This doesn't exclude the null cases, I think it'd be better to use a builder and rely on the `AddIfNotNull` method.